libs/design-system/.storybook/preview-styles/index.scss

```diff
@@ -5,3 +5,7 @@
 html {
   font-family: Metropolis;
 }
+
```

Comment:
You'll have to move this once you merge develop. You can move it into `design-system/styles/_resets.scss`